pages/common/rsync.md

```diff
@@ -19,6 +19,10 @@
 
 `rsync -r {{remote_host_name}}:{{remote_folder_location}} {{local_folder_location}}`
 
+- Transfer directory contents (but not the directory itself) from a remote into local:
```

Comment:
What do you mean by `into` local here? Is this "into a local directory?

Reply:
This was a mistake... I've now replaced "into" with "to", so it matches the language of the example above it.